
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264953162 -3600
# Node ID 923ef40994edbc80c6d9308b05b90cfabee984d0
# Parent  17615367bcb7bebe8fd23b7de1802f891b747950
libv4l: enable whitebalance by default for SN9C325 + OM6802 based cams

From: Hans de Goede <hdegoede@redhat.com>

libv4l: enable whitebalance by default for SN9C325 + OM6802 based cams

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Sun Jan 31 09:59:50 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Sun Jan 31 16:52:42 2010 +0100
@@ -288,6 +288,8 @@
   { 0x0c45, 0x6001, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110C */
   { 0x0c45, 0x6005, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110C */
   { 0x0c45, 0x6007, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110D */
+  /* sn9c105 / sn9c120 based devices (sonixj) */
+  { 0x0c45, 0x6128, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OM6802 */
   /* Pac207 based devices */
   { 0x041e, 0x4028, 0,    NULL, NULL, V4LCONTROL_WANTS_WB, 1500 },
   { 0x093a, 0x2460, 0x1f, NULL, NULL, V4LCONTROL_WANTS_WB, 1500 },

